lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -303,15 +303,14 @@ def add_reference(post, authorization)
         post[:transaction][:reason] = 'n/a'
       end
 
-      def add_payment_method_to_be_tokenized(post, payment_method)
+      def add_payment_method_to_be_tokenized(post, payment_method, options)
         credit_card_token = {}
-        credit_card_token[:payerId] = generate_unique_id
+        credit_card_token[:payerId] = options[:payer_id] || generate_unique_id
         credit_card_token[:name] = payment_method.name.strip
-        credit_card_token[:identificationNumber] = generate_unique_id
+        credit_card_token[:identificationNumber] = options[:dni_number]
```

Comment:
In fact I'm not seeing either `payerId` or `identificationNumber` fields in their documentation at all; they may have changed since, or erroneously added, to the initial implementation. I assume your remote test passes with this, but honestly their sandbox is extremely permissive, so I'm hesitant to take it as a confirmation.
Could you link to the documentation you're using?

Reply:
I was using their tokenization documentation: http://developers.payulatam.com/en/api/tokenization.html

The remote test for store is passing.

```
Unit Tests:
23 tests, 89 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote Tests:
19 tests, 49 assertions, 2 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
89.4737% passed
```
The failures in remote tests are for void and capture, these fails because an internal problem in Payu Latam sandbox.

Reply:
Please merge this PR, it correctly fixes the `store` method is which is currently broken